srcpkgs/squeezelite/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'squeezelite'
+pkgname=squeezelite
+version=1.9.6.1198
+revision=1
+_githash=1c1f429ae9494748fe0fa5807c3a22710edaa12f
```

Comment:
Tagged releases are preferred.

Reply:
They don't seem to tag stuff, eventhough versioned releases exist.

That said, seems like it's been updated to 1.9.7.1256 https://sourceforge.net/projects/lmsclients/files/squeezelite/linux/

I can't find which version was used to build those fragments, which is kind of suspect.

Reply:
I think that works. Bad upstream, but it's what we got.

Reply:
i've update to the latest one. Can't do much with such upstream.